Makefile

```diff
@@ -8,27 +8,44 @@ endif
 
 .PHONY: dev
 dev: ## Installs dbt-* packages in develop mode along with development dependencies.
-	pip install -r dev-requirements.txt -r editable-requirements.txt
+	@\
+	pip install -r dev-requirements.txt -r editable-requirements.txt && \
+	pre-commit install
 
 .PHONY: mypy
 mypy: .env ## Runs mypy for static type checking.
-	$(DOCKER_CMD) tox -e mypy
+	@\
+	$(DOCKER_CMD) pre-commit run mypy | grep -v "[INFO]"
 
 .PHONY: flake8
 flake8: .env ## Runs flake8 to enforce style guide.
-	$(DOCKER_CMD) tox -e flake8
+	@\
+	$(DOCKER_CMD) pre-commit run flake8 | grep -v "[INFO]"
+
+.PHONY: black
+black: .env ## Runs black to enforce style guide.
+	@\
+	$(DOCKER_CMD) pre-commit run black | grep -v "[INFO]"
 
 .PHONY: lint
 lint: .env ## Runs all code checks in parallel.
```

Comment:
Good question.  I thought it was a makefile thing, but I may be mistaken.  I'll research and get back to ya.

Reply:
So this ends up being a bit of a big question.  Here's the TL;DR--

- Make, pre-commit, and the tools themselves all support some sort of parallelism... to get the best performance here we want to use exactly _one_ method.
- Pre-commit's parallelism is based on splitting the work to be done based on batching requests to each hook based on the number of files.  This makes the most sense to optimize for since file batch control is one of the best features pre-commit has.
- I've confirmed that it works.. PStree output showing black doing it's thing below.
- All that said, if it's not fast enough, it's not fast enough so I'm removing it from the PR.